haralyzer/assets.py

```diff
@@ -324,7 +332,16 @@ def filter_entries(self, request_type=None, content_type=None,
             if http_version is not None and not p.match_http_version(
                     entry, http_version, regex=regex):
                 valid_entry = False
-
+            if source is not None and source == 'internal':
+                if p.hostname == get_entry_hostname(entry):
+                    valid_entry = True
```

Comment:
Assigning it to True is not necessary, this is because all entries start off with `valid_entry = True`. So, we only need to have the appropriate check that would set it to False.